File: src/main/java/com/neutralplasma/virtusbot/commands/PlayerCommand.java
package com.neutralplasma.virtusbot.commands;

import com.jagrosh.jdautilities.command.Command;
import com.neutralplasma.virtusbot.VirtusBot;

public abstract class PlayerCommand extends Command {

    public PlayerCommand()
    {
        this.category = new Category("Player", event -> !VirtusBot.getBlackList().isBlackListed(event.getAuthor().getId()));
        this.guildOnly = true;
    }

}

File: src/main/java/com/neutralplasma/virtusbot/commands/PlayerCommand.kt
package com.neutralplasma.virtusbot.commands

import com.jagrosh.jdautilities.command.Command
import com.jagrosh.jdautilities.command.CommandEvent
import com.neutralplasma.virtusbot.VirtusBot.blackList
import java.util.function.Predicate

abstract class PlayerCommand : Command() {
    init {
        this.category = Category("Player", Predicate { event: CommandEvent -> !blackList!!.isBlackListed(event.author.id) })
        guildOnly = true
    }
}